docs/sql/array.md

```diff
@@ -47,6 +47,7 @@ as `CREATE TABLE data AS (CITIES VARCHAR ARRAY, COUNTRY VARCHAR)`.
 | `ARRAY_LENGTH(` _array_ `)`                                          | Another name for `CARDINALITY`                                                                                                                                                                                                            | `ARRAY_LENGTH( ARRAY [1, 2, 3])` => `3`                             |
 | `ARRAY_MAX(` _array_ `)`                                             | Returns the maximum value in the _array_                                                                                                                                                                                                  | `ARRAY_MAX(ARRAY [9, 1, 2, 4, 8, null])` => 9                       | 
 | `ARRAY_MIN(` _array_ `)`                                             | Returns the minimum value in the _array_. Ignores null values.                                                                                                                                                                            | `ARRAY_MIN(ARRAY [9, 1, 2, 5, 8, null])` => 1                       |
+| `ARRAYS_OVERLAP(` _array1_ `, ` _array2_ `)`                         | Returns `NULL` if _array1_ or _array2_ is `NULL`. Returns true if the two arrays have at least one element in common. Returns `NULL` if either of the arrays have a `NULL` element and no common element. Returns false otherwise.        | `ARRAYS_OVERLAP(ARRAY [1, 2, 3], ARRAY [3, 4, 5])` => true          |
```

Comment:
really? if there is any null element the result is null?
that's a terrible function.

Reply:
If there is any null element and no common element, the result is null.